src/p5xr/core/p5xrInput.js

```diff
@@ -63,6 +64,18 @@ export default class p5xrInput {
     const p = this._pose?.transform?.position;
     return new p5.Vector(p.x, p.y, p.z);
   }
+  /** @returns {p5.Vector} Returns the current rotation as a Vector */
+  get rotation() {
+    this.updatePose();
+    if (this._pose){
+      const {x, y, z, w} = this._pose?.transform?.orientation;
+      let q = new Quaternion(x, y, z, w);
+      let e = q.toEuler();
+      let c = (p5.instance.angleMode == RADIANS)? 1 : 180 / Math.PI; // TODO: Change output with ANGLE MODE
```

Comment:
Make sure to run `npm run grunt` locally to check for linting issues before push. Here you are using the `isLooselyEqual` comparison operator vs the `isStrictlyEqual` one which is against the linting rules. [Here is a reference on the difference.](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness)

Reply:
Thank you!